helper.py

```diff
@@ -1,12 +1,338 @@
+                        if c == False:
+                            condition = True 
+                    
+            else:
+
+                if type3 == True:
+
+                    if type4 == True:
+                        length = int(input("How many characters do you want in your password? "))
+                        sample_space = string.punctuation + string.digits
+                        password = ''.join((random.choice(sample_space) for i in range(length)))
+                        clipboard.copy(password)
+                        clear_screen()
+                        print('''--------------------------------------------------------------------------------------------------------------
+Your password is:''', password, '''
+--------------------------------------------------------------------------------------------------------------''')
+                        c = cli_ui.ask_yes_no("Do you like this password? ")
+
+                        if c == True:
+                            clear_screen()
+                            print("The password has been copied to your clipboard.")
+                            condition = False
+                            return password
+                        if c == False:
+                            condition = True 
+                        
+                    else:
+                        length = int(input("How many characters do you want in your password? "))
+                        sample_space = string.punctuation 
+                        password = ''.join((random.choice(sample_space) for i in range(length)))
+                        clipboard.copy(password)
+                        clear_screen()
+                        print('''--------------------------------------------------------------------------------------------------------------
+Your password is:''', password, '''
+--------------------------------------------------------------------------------------------------------------''')
+                        c = cli_ui.ask_yes_no("Do you like this password? ")
+
+                        if c == True:
+                            clear_screen()
+                            print("The password has been copied to your clipboard.")
+                            condition = False
+                            return password
+                        if c == False:
+                            condition = True 
+
+                else:
+                    if type4 == True:
+                        length = int(input("How many characters do you want in your password? "))
+                        sample_space = string.digits
+                        password = ''.join((random.choice(sample_space) for i in range(length)))
+                        clipboard.copy(password)
+                        clear_screen()
+                        print('''--------------------------------------------------------------------------------------------------------------
```

Comment:
You can probably create a helper that prints out this line of `-`s because it makes the long line of code looks odd